base_station/gui/src/components/LCMEcho.vue

```diff
@@ -142,7 +141,9 @@
           {'topic': '/tennis_ball', 'type': 'TennisBall'},
           {'topic': '/imu', 'type': 'IMU'},
           {'topic': '/gps', 'type': 'GPS'},
-          {'topic': '/microscope', 'type': 'Microscope'}
+          {'topic': '/microscope', 'type': 'Microscope'},
+          {'topic': '/zed_gimbal_command', 'type': 'ZedGimbalPosition'},
+          {'topic': '/zed_gimbal_position', 'type': 'ZedGimbalPosition'}
```

Comment:
change this to zed_gimbal_data, and ZedGimbalData **as per the ESW auton nav ICD**